CHANGELOG.md

```diff
@@ -31,6 +31,7 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 - Exports components and variables that were initially missed when they were added: `ColorVariants`, `Fieldset`, `IconAlign`, `StatusBadgeTypes`,
 - Fixes `SearchBar` by passing necessary props down to its `TextInput` through the `textInputProps` prop.
 - Fixes `DatePicker` component unit tests.
+- Fixes the Github Actions "CI / Lint" task by pinning the Typescript version.
```

Comment:
What not wrap `CI / Lint` in backticks?

Reply:
I can, I typically leave it for component or variable names.